subtree-splits/blt-project/phpcs.xml.dist

```diff
@@ -34,9 +34,5 @@
   <exclude-pattern>*/behat</exclude-pattern>
   <exclude-pattern>*/node_modules</exclude-pattern>
   <exclude-pattern>*/vendor</exclude-pattern>
-  <!-- Exclude files provided by ACSF that fail code sniffing. -->
```

Comment:
These exclude patterns were only added a previous workaround to these issue. Note that directories such as `factory-hooks` are not included in the `file` directives above, and thus will continue to be excluded now that this bug is fixed.